frame/nomination-pools/src/lib.rs

```diff
@@ -1172,6 +1172,9 @@ pub mod pallet {
 	use frame_support::traits::StorageVersion;
 	use frame_system::{ensure_signed, pallet_prelude::*};
 
+	#[cfg(feature = "try-runtime")]
+	use frame_support::traits::TryStateSelect;
```

Comment:
I usually use the full path to avoid feature gated imports.

Reply:
The mod `try_runtime` is private.

Reply:
Everything should be re-exported from `frame-support::traits`